reference/dic_tags.rst

```diff
@@ -378,6 +389,11 @@ the :class:`Symfony\\Component\\HttpKernel\\CacheWarmer\\CacheWarmerInterface` i
         }
     }
 
+.. deprecated:: 5.1
+
```

Comment:
Same, is the « the » needed/correct with plural ? Asking a question :)

Reply:
Apparently this is easy ... but in practice it's not. See https://english.stackexchange.com/questions/10854/rule-on-absence-of-the-article-the-with-plural-nouns. I'd keep the current `the` ... but native speakers can remove it if needed in future PRs. Thanks.